connect/client/models/iterators.py

```diff
@@ -0,0 +1,139 @@
+from connect.client.utils import get_values, parse_content_range
+
+
+class aiter:
```

Comment:
Could you explain quickly why do we need that?

Reply:
If the ResourceSet has already been evaluated (i.e. all the pages of results have been loaded), it should be possible to iterate again over results, so this iterator is needed to iterate over a list of results cached inside this ResourceSet.